src/app/api/residents/bulk/route.ts

```
@@ -109,11 +116,11 @@
~        // Soft delete residents
~        const { error: deleteError } = await supabaseAdmin
~          .from('residents')
~          .update({
             is_active: false,
             updated_at: new Date().toISOString(),
             updated_by: user.id,
-          })
+          } as Record<string, any>)
           .in('id', resident_ids);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)